k-distribution/tests/regression-new/post-process/Makefile

```diff
@@ -0,0 +1,32 @@
+VENV_DIR := venv
+ACTIVATE := . $(VENV_DIR)/bin/activate
+VENV     := $(VENV_DIR)/pyvenv.cfg
+
+KOMPILED_DIR := kompiled
+
+
+.PHONY: $(VENV_DIR)
+
+
+# kompile
+
+$(KOMPILED_DIR)/timestamp: assign.k transform.py $(VENV)
+	   $(ACTIVATE)                  \
+	&& kompile assign.k             \
+	   --backend haskell            \
+	   --output-definition kompiled \
+	   --emit-json                  \
+	   --post-process 'python3 transform.py'
+
+
+# virtualenv
+
+$(VENV):
+	   virtualenv -p python3.8 $(VENV_DIR) \
```

Comment:
This line specifies the python version which is not stable on all machines.
For example, I have 3.10
It's going to be really annoying to change it to test on my local machine.
Can we do something about it?
This holds true for the test in the main pyk folder as well.

Reply:
Done.